packages/react-components/src/components/Notifications.tsx

```diff
@@ -28,47 +34,32 @@ export interface NotificationsProps {
    * @defaultValue 2
    */
   maxNotificationsToShow?: number;
-}
 
-/**
- * All strings that may be shown on the UI in the {@link Notifications}.
- *
- * @beta
- */
-export interface NotificationsStrings {
   /**
-   * Unable to reach Chat service.
-   *
-   * This can mean:
-   *   - Incorrect Azure Communication Services endpoint was provided.
-   *   - User's network connection is down.
+   * Callback called when the dismiss button is triggered.
+   * Use this to control notifications shown when they dismissed by the user.
+   * Note this onDismiss function will affect all notifications in the same stack
    */
-  unableToReachChatService: NotificationBarStrings;
+  onDismissNotification?: (dismissedNotifications: ActiveNotification) => void;
 
   /**
-   * User does not have access to the Chat service.
-   * This usually means that either the Azure Communication Services endpiont or the token provided are incorrect.
-   */
-  accessDenied: NotificationBarStrings;
-
-  /**
-   * User is no longer on the thread.
+   * If set, notifications with {@link ActiveNotification.timestamp} older than the time this component is mounted
+   * are not shown.
    *
-   * See also: {@link NotificationsStrings.sendMessageNotInChatThread} for a more specific error.
-   */
-  userNotInChatThread: NotificationBarStrings;
```

Comment:
Where did all these chat strings go?

Reply:
We are not using the new notification for chat so removing the chat strings